src/app/client/src/app/modules/program-dashboard/components/program-datasets/program-datasets.component.ts

```diff
@@ -313,6 +313,7 @@ export class DatasetsComponent implements OnInit, OnDestroy {
     this.newData = false;
     this.noResult = false;
     this.districts = this.organisations = []
+    this.userAccess = true;
```

Comment:
when is the value becomes false
this.userAccess

Reply:
When the particular user is a program manager and that program has been enabled for program level reports (new feature)

Reply:
after selecting a resource this is made true as both program manager and designer have access to resource/solution level reports and csvs